### PR TITLE
Attemps to fix the tesla lag by making use of our lord and savior, prob(80)

### PR DESCRIPTION
**Pull request by alexkar598**
# General Documentation

### Intent of your Pull Request
Machines will only explode 20% of the time instead of 85% of the time now, they can also get hyper damaged 20% of the time instead of 15% and they just get ex_act'd devastate 60% of the time.

### Why is this change good for the game?
Explosion particle effects are laggy AF, less explosions = less lag

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Machines will only explode 20% of the time instead of 85% of the time now, they can also get hyper damaged 20% of the time instead of 15% and they just get ex_act'd devastate 60% of the time.

### What should players be aware of when it comes to the changes your PR is implementing?
Machines will only explode 20% of the time instead of 85% of the time now, they can also get hyper damaged 20% of the time instead of 15% and they just get ex_act'd devastate 60% of the time.

### What general grouping does this PR fall under? 
Tesla

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Machines will only explode 20% of the time instead of 85% of the time now, they can also get hyper damaged 20% of the time instead of 15% and they just get ex_act'd devastate 60% of the time.
# Changelog
:cl:  
rscadd: Machines will only explode 20% of the time instead of 85% of the time now, they can also get hyper damaged 20% of the time instead of 15% and they just get ex_act'd devastate 60% of the time.
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Did a rare thing and tested my changes in game for once, time dilation was 5% on old tesla and 2% on new tesla.

Also the station is much more derelic with my changes instead of the old tesla which just leaves no station behind
Video: [https://streamable.com/kn68tc](https://streamable.com/kn68tc)
Old: before
New: after

**Comment by Yogbot-13** [comment]
#### Test Merge Deployed

##### Server Instance
main

##### Revision
Origin: fff1770e8b544be509edef7c14900647e0afebc6
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 47528dc0e18249af8fea62ba7c0d7339a00763f3

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: fff1770e8b544be509edef7c14900647e0afebc6
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 02583086b1247c9a1be51165afc516aad0e5e7a7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 17d3b574ef653d3278c26a59dc8418bee32d4849
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 3dc4353d50b708e98f7a70c983f21bd3009239e1

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: b282edb69de8b2d8900b7573ebac641561d392d3
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 461b198c2ad376e0a8dcba584d5cabc8cd93b91d

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f1ccd63a36b575f6b21d51678f597a016dea4341
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 9ae5169f85f4615a97cfce794663b0fcd3630291

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f1ccd63a36b575f6b21d51678f597a016dea4341
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: ff51cbb8edef33f36a4b21e1679264ce72f404e8

**Comment by alexkar598** [comment]
Change was tested live by spawning a tesla on the station after everyone leaves, at most, the lag was at 24% while it was sitting at 20% without the tesla. Id say this was very successful

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 77b18567de776b91f70821b0cb89efcea50e5638
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 74d901727666ecdb4a83807850b62b7ae4596cc2

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: d9fa53dbf20373162cd911ee6c408579cbb08dd4
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 71059923354c07cd59b391346f037e2cc0a88638

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: d9fa53dbf20373162cd911ee6c408579cbb08dd4
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: e241a029bff0a862598d4cf5330270ab7a61901c

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: ef0db3a25ba81dee1c39ea8fc8162a7a4a752ce0
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: aaf2a4de3a62a53112d314c3f9281454f06ce656

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 61be296c7775d9ea88e9b8a729456b211e4cc029
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 7556b2e6eebbdd8cd977546ea7280d0bb63bd23f

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 61be296c7775d9ea88e9b8a729456b211e4cc029
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 30cee7883419fedc2b14d05329f2362084d4a522

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: a3380909465cd8b2082b69723d93038188c356e5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 952eb07ef4c61276755bc1d018d27c948afb0e36

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 9a2ec9b6425c6d90690405f9c3b94f4666402c94

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: dd1d7fc33764ab1f7a35cf63a176bb8ea99c5bb7
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 14cdaf78532bb345dde5558d0304548c1bc44f79

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: dd1d7fc33764ab1f7a35cf63a176bb8ea99c5bb7
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 38094ac42f2645436ff266e4acf28dbe0a1101bc

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: a6e2415c469ba1af6e29b54aa239ed65b8c36ee7
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: d24b215e0e8c6c5423888ae49e4f29a9cf171b3c

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 66b7150aeab900c3f31c14b1bd1e42b0bf923c18
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: ef2e93fba57b1a0477b759c6d1a9a04f2aa0c565

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 158c00ddf79f3de3a5da5964b55ac17a89978863
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: b8c32c664435221e3bc7144827e3654e90e4e6dc

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: ca59602e0478d0927aad7a9b1fd4711bde1f9c82
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: c2377bfb63991a30acd46b8f81123ed6b04133b5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: 576b980d7860d3a2d8af75956984af14a64580ef

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: dba8d9be2e90bc065bfbc1306bff9d3d3f9529aa
Server: ce13dee3d6e20b89dc7daedfb859ee58550b7dcf
